libraries/AC_SafeRTL/AC_SafeRTL.h

```diff
@@ -0,0 +1,146 @@
+#pragma once
+
+#include <AP_Buffer/AP_Buffer.h>
+#include <AP_HAL/AP_HAL.h>
+#include <AP_Math/AP_Math.h>
+#include <AP_AHRS/AP_AHRS.h>
+#include <DataFlash/DataFlash.h>
+#include <GCS_MAVLink/GCS.h>
+
+#include <bitset>
+#include <vector>
+
+// definitions and macros
+#define SAFERTL_ACCURACY_DEFAULT        2.0f    // default _ACCURACY parameter value.  Points will be no closer than this distance together.
```

Comment:
added units to comments.  normally when we use SI units we don't specify the units in the name.